1.8/faq/database-errors.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: page
+title: "Fixing Database errors"
+categories: [faq]
+---
+
+MyBB set-up is made to be as simple as possible, however sadly sometimes there are a few issues that can occur. Including, it seems, database connection issues.
+With this FAQ tutorial, we will go over how to fix common database issues.
+
+Note: Please find the set-up installation guide for MyBB here: http://docs.mybb.com/1.8/install/
+
+# Access denied for user
+Getting an error like this?:
+![alt text](https://gyazo.com/0a96164ea30ad8513a7553a98ad036ac "Access Denied database")
+
+This means that your configuration is wrong! Don't worry though, this is easy to fix. 
+First - lets find out our username and password.
+
+If you're on a webhost environment, login to your CPanel and select your database configuration. 
+![alt text](http://puu.sh/qmkpY/2c24c1ec6f.png "CPanel database account")
+
+Create a username and password, and then assign this user to your database:
+![alt text](http://puu.sh/qmkvf/d9763cebab.png "Assign user to database")
+
+Now that you have assigned your user to your database, we need to change our MyBB configuration to reflect this.
+Open your preferred FTP software and navigate to '/inc/config.php'
+Enter the details that you made earlier like so:
+
+![alt text](http://puu.sh/qmkK7/8ca656a3a5.png "Database settings")
+
+# Access denied cont.
+Another cause for the above issue is if your database is hosted seperately to your website. In this case, you will need to enable remote database connection.
+On a Unix environment, you will want to login via SSH and enter your MySQL configuration
+
+Ubuntu:
+```
+mysql -u (YOUR USER) -p
+```
+You will then need to enter your password
+
+You can then grant permissions for a user on a specific IP address, or % for wildcard (all IP addresses)
+For example:
+```
+CREATE USER 'USERNAME HERE'@'IP ADDRESS HERE' IDENTIFIED BY 'PASSWORD HERE';
+```
+
+```
+GRANT ALL PRIVILEGES ON * . * TO 'USERNAME HERE'@'IP ADDRESS HERE';
+```
+The asterisks in this command refer to the database and table (respectively) that they can access—this specific command allows to the user to read, edit, execute and perform all tasks across all the databases and tables.
+
+Once you have finalized the permissions that you want to set up for your new users, always be sure to reload all the privileges.
+```
+FLUSH PRIVILEGES;
+```
+
+#Table <table> doesn't exist!

```

Comment:
`<table>` is a HTML tag. If anyone acknowledged wth HTML reads this, they may be confused. It has a special meaning for developers and you wanted to emphasise that it may be "insert anything here", which is usually done with `[]`: `[table]`